influxdb3_write/src/write_buffer/validator.rs

```diff
@@ -226,6 +214,10 @@ impl WriteValidator<WithCatalog> {
     }
 }
 
+/// Vec storing qualified column data from an incoming write, the bool designates new columns added
```

Comment:
what bool?

Reply:
Ah, good catch. An artefact of a failed attempt that one. I will clean that up.

Reply:
Fixed in https://github.com/influxdata/influxdb/pull/25495/commits/8b75c1d499040b9a94e5d52aa2be3e328e1d55e4